pkg/kubectl/cmd/util/clientcache.go

```diff
@@ -53,20 +53,18 @@ type ClientCache struct {
 
 	matchVersion bool
 
-	defaultConfigLock sync.Mutex
-	defaultConfig     *restclient.Config
+	mutex         sync.Mutex
```

Comment:
No issue changing it but I was trying to be consistent with the package.
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/util/cached_discovery.go#L50-L51